integrationtest/ValidationProject/ValidateStrykerResults.cs

```diff
@@ -11,6 +15,25 @@ namespace IntegrationTests
 {
     public class ValidateStrykerResults
     {
+        private readonly ReadOnlyCollection<SyntaxKind> _blacklistedSyntaxKindsForMutating =
```

Comment:
Please add whitelisting for nodes inside of a MethodDeclarationSyntax or PropertyDeclarationSyntax

Reply:
Will do, we should also add SyntaxKind.ConstructorDeclaration and SyntaxKind.FieldDeclaration to the whitelist.